.github/pull_request_template.md

```diff
@@ -1,3 +1,14 @@
+## Important: Feature freeze / release candidate phase for Betaflight 3.3
+
+From 01/02/2018 until the release of Betaflight 3.3.0 (scheduled for 01/03/2018), the project is in a 'feature freeze / release candidate' phase. This means:
+
+1.) Pull requests can still be submitted as normal. Comments / discussions will probably be slower than normal due to shifted priorities;
```

Comment:
if you remove the `)` from these it should turn it into a markdown <ol>

Reply:
Good point, thanks. In saying that, the value of using markdown in these 'templates' is limited, since the user sees them pre-filled into an edit box, and we explicitly ask them to remove the text before submitting. ;-)